### ISSUE TITLE
WebTorrent Maintainers Call

### ISSUE THREAD

**Issue by feross** [webtorrent/webtorrent#1906]
Hi everyone,

I’d like to start a monthly maintainer's call to discuss PRs that need to be reviewed and whatever is on people’s minds. I know there are PRs which are blocked waiting for my review and I think a regular call would help to surface these.

Also, I think it would be fun to talk with all of you!

I'm proposing that we meet the first Friday of the month at 10am Pacific Time, which I believe should be good for North/South America and Europe timezones (where I believe most of us live). So, the first meeting time would be:

Friday, September 4 @ 10am PT
Meeting URL: https://meet.google.com/pjw-yory-ccc

How does this sound?

**Comment by ftreesmilo** [comment]
I'd love to attend, but it's far too early for me.
Is it possible you could take a look at my 2 prs?   #1900 (very small) and #1903 (a bit more involved)

**Comment by feross** [comment]
Thanks to @alxhotel @DiegoRBaquero @Borewit @dsernst for attending today!

I made a calendar that anyone can subscribe to if you want to attend a future maintainers call:

- Add to Google Calendar: https://calendar.google.com/calendar/embed?src=h9fkfc4k3rnbuvch61cflv6l88%40group.calendar.google.com&ctz=America%2FLos_Angeles
- Download .ical file: https://calendar.google.com/calendar/ical/h9fkfc4k3rnbuvch61cflv6l88%40group.calendar.google.com/public/basic.ics

**Comment by feross** [comment]
Next maintainers call happening at Friday, November 6 at 8:00am!

**Comment by feross** [comment]
Next maintainers call is December 4 at 10am PT.

**Comment by ftreesmilo** [comment]
next call?

**Comment by feross** [comment]
Hi all! When we started the maintainer call, my goal was for us to discuss the roadmap and check in on the progress towards these goals. In my experience, it hasn't been super useful for this purpose. It's been pretty hard to find a time that works for everyone who is interested in attending (for example, the meeting time hasn't worked for me for the past 3 meetings). So, the other maintainers and I have decided to put the call on hold for the time being and move all discussion to GitHub where everyone can participate asynchronously. We can revisit this decision in the future if circumstances change.
